### ISSUE TITLE
Issue nxos_vxlan_evpn_l3vni.jinja2 Version: 1.0.3

### ISSUE THREAD

**Issue by btr1975** [btr1975/QuickConfigTemplates#8]
```
{%  if vxlan_evpn_l3vni.l2_vni_id == vxlan_evpn_l3vni.l3_vni_id %}
```
should be 
```
{%  if vxlan_evpn_l3vni.l2_vni_id != vxlan_evpn_l3vni.l3_vni_id %}
```

**Comment by btr1975** [comment]
Need to rework the vlan portion, no evpn

**Comment by btr1975** [comment]
Fixed in commit 782633c877a6c83db9e91ce352554e1e84b2d957
